pycbc/pnutils.py

```diff
@@ -472,7 +472,11 @@ def get_final_freq(approx, m1, m2, s1z, s2z):
     "SEOBNRv2RD"  : lambda p: get_freq("fSEOBNRv2RD", p["m1"], p["m2"],
                                               p["s1z"], p["s2z"]),
     "SEOBNRv2Peak": lambda p: get_freq("fSEOBNRv2Peak", p["m1"], p["m2"],
-                                              p["s1z"], p["s2z"])
+                                              p["s1z"], p["s2z"]),
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->